infrastructure/variables.tf

```diff
@@ -18,3 +18,139 @@ variable "env" {
 variable "common_tags" {
   type = "map"
 }
+
+// health service urls
+variable "health_ccd_component_api" {}
```

Comment:
didn't @ernestman28 just remove these?

Reply:
@ernestman28  please confirm

Reply:
@paddybasi @udaydenduluri33 my earlier PR was for MO not for MC. but Paddy right, we don't need those files. I guess that the issue with cherry picking, it seems this commit never went to `master` https://github.com/hmcts/rpx-xui-webapp/commit/1dae3fb91eecff8f673da7def6744a4c9f505ebe

Reply:
but if easier, just merge it back first and I will tidy up later.

Reply:
Thanks @ernestman28

Reply:
actually, I think @ernestman28 it has gone in, but this PR seems to be bringing these changes back? so yeah probably an issue with cherry picking, either way we don't really need these @udaydenduluri33